torch/csrc/jit/pickler.cpp

```diff
@@ -415,17 +415,11 @@ void Pickler::pushDouble(double value) {
 void Pickler::pushLong(const std::string& data) {
   uint64_t size = data.size();
 
-  if (size <= std::numeric_limits<uint8_t>::max()) {
-    push<PickleOpCode>(PickleOpCode::LONG1);
-    push<uint8_t>(size);
-  } else {
-    TORCH_INTERNAL_ASSERT(
-        data.size() > std::numeric_limits<uint32_t>::max(),
-        "Cannot pickle a long with a size larger than 4 bytes")
-    push<PickleOpCode>(PickleOpCode::LONG4);
-    // TODO: should this be uint32_t instead?
-    push<uint64_t>(size);
-  }
+  TORCH_INTERNAL_ASSERT(
+    size <= std::numeric_limits<uint8_t>::max(),
+    "Cannot pickle a long with a size larger than 256 bytes");
```

Comment:
This should say `Cannot pickle a long with a size larger than 1 byte`

Reply:
I can change this message, but given that data.size() can be up to 256, saying that the long's size can't be larger than 1 byte doesn't seem accurate.

I could say the more-awkward-but-accurate:
    "Cannot pickle a long with the size of size is larger than 1 byte"

Or, is there less-awkward-but-also-accurate preferred version of the assert text?

Reply:
Maybe just `Cannot pickle a long larger than 255 bytes` (0 is a valid size, 256 is not)

Reply:
fixed, thanks!